src/dom/Struct.ts

```diff
@@ -62,6 +62,7 @@ export class Struct extends Value(Object, IonTypes.STRUCT, FromJsConstructor.NON
             },
             deleteProperty: function (target, name): boolean {
                 // Property is deleted only if it's in _field Collection
+                // reference Document: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#Return_value
                 if (name in target._fields) {
                     delete target._fields[name];
                 }
```

Comment:
Because this comment explains the meaning of the return value, I think it would be better to have it immediately above the `return true` on line 69.

```suggestion
                if (name in target._fields) {
                    delete target._fields[name];
                }
                // Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#Return_value
```

If you agree with this suggestion, note that you can accept the change directly from the GitHub UI using the "commit suggestion" button above. You don't need to make another commit and push it.